pkgs/development/tools/build-managers/boot/builder.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
```

Comment:
Could you please remove this line? ```/bin/bash``` doesn't exist on e.g. NixOS:

```
$ /bin/bash
bash: /bin/bash: No such file or directory
```
:smile: 

Builds scripts will be evaluated by ```bash```. It's the default behavior, so we don't need a shebang in this case.